iina/Extensions.swift

```diff
@@ -321,6 +321,14 @@ extension Data {
   }
 }
 
+extension String {
```

Comment:
Filenames for mpv's watch_later file are calculated by [configfiles.c#L166](https://github.com/mpv-player/mpv/blob/abbc8fc84a99a739e2566e4f7e022b3c1bdde1f3/player/configfiles.c#L166), so I have to implement a md5 method for string in order to get the corresponding watch_later file.